src/OpalCompiler-Core/Behavior.extension.st

```diff
@@ -104,3 +104,29 @@ Behavior >> sourceCodeTemplate [
 	| temporary variable names |
 	statements'
 ]
+
+{ #category : #'*OpalCompiler-Core' }
+Behavior >> sourceCodeTemplateFor: aClassOrMetaclass [ 
+	"Answer an expression to be edited and evaluated in order to define 
+	methods in this class or trait."
+
+		^ aClassOrMetaclass isMeta 
+				ifTrue: 
+[ 
+'messageSelectorAndArgumentNames
+	"comment stating purpose of factory/class-side message"
```

Comment:
why factory? The comments should be different for traits

Reply:
> why factory?

The idea that "classes are objects" is foreign to most newcomers.  Early on I read that Smalltalk comes with the Factory Design Pattern "built-in" and that really had an impact on me.  Framing the unfamiliar against the familiar provide better learning outcomes.  But I don't see this mentioned much and think we under promote this.  
This was just a slight attempt to promote that.  But its a personal view, so if you don't like it, I can delete "factory"  from  it.

> The comments should be different for traits

Can you suggest some code to test for this, plus associated text?

Reply:
Yes - please delete the "factory"